### PR TITLE
Added support for multiple previous contracts

### PR DESCRIPTION
**Pull request by tomarnold-ccpo**

### CODE REVIEW

File: f600233d1b154d507b782f84604bcb12/update/sys_script_include_3d46262b97a751106fa8b4b3f153af90.xml [hunk 1]
```diff
@@ -104,7 +105,14 @@ Dow.prototype = {
 		"PORTABILITY_PLAN", "ADVISORY_ASSISTANCE", "HELP_DESK_SERVICES", "TRAINING",
 		"DOCUMENTATION_SUPPORT", "GENERAL_CLOUD_SUPPORT"
 	],
-
+	//  CURRENT_CONTRACT_INFO_FIELDS: [
+	//  	"acquisition_package", "business_size", "competitive_status", "contract_number",
+	//  	"contract_order_expiration_date", "contract_order_start_date", "current_contract_exists",
+	//  	"incumbent_contractor_name", "task_delivery_order_number"
+	//  ],
```
Comment (jasonburkert-ccpo) [inline f600233d1b154d507b782f84604bcb12/update/sys_script_include_3d46262b97a751106fa8b4b3f153af90.xml:112]: Please remove commented code if not needed

File: f600233d1b154d507b782f84604bcb12/update/sys_script_include_3d46262b97a751106fa8b4b3f153af90.xml [hunk 2]
```diff
@@ -987,11 +995,70 @@ Dow.prototype = {
...
+			return {
+				// contractNumber: cci.contract_number.toString(),
+				// contractOrderExpirationDate: ctb(cci.contract_order_expiration_date.toString()),
+				// contractOrderStartDate: ctb(cci.contract_order_start_date.toString()),
+				// incumbentContractorName: cci.incumbent_contractor_name.toString(),
+				// contractExists: ctb(cci.current_contract_exists.toString()),
+				// taskDeliveryOrderNumber: cci.task_delivery_order_number.toString(),
+				// competitiveStatus: ctb(cci.competitive_status.toString()),
+				// businessSize: ctb(cci.business_size.toString()),
+				// acquisitionPackage: cci.acquisition_package.toString()
```
Comment (jasonburkert-ccpo) [inline f600233d1b154d507b782f84604bcb12/update/sys_script_include_3d46262b97a751106fa8b4b3f153af90.xml]:
```suggestion
```

File: f600233d1b154d507b782f84604bcb12/update/sys_script_include_3d46262b97a751106fa8b4b3f153af90.xml [hunk 3]
```diff
@@ -987,11 +995,70 @@ Dow.prototype = {
...
 			default:
 				gs.warn("Table " + table + " is not known to be associated with a package.");
 				return [];
 		}
 
+	},
+		/**
+	 * Gather a single contract record
+	 * @param {GlideRecord} contract - a contract record
+	 * @return {object} a contract record
```
Comment (jasonburkert-ccpo) [inline f600233d1b154d507b782f84604bcb12/update/sys_script_include_3d46262b97a751106fa8b4b3f153af90.xml]: It's not clear to me what this is doing.  Can expand about what this does and why?

**Review by jasonburkert-ccpo — CHANGES_REQUESTED** [review]
The rest looks good

File: f600233d1b154d507b782f84604bcb12/update/sys_script_include_3d46262b97a751106fa8b4b3f153af90.xml (hunk 1 above)
Comment (tomarnold-ccpo) [reply f600233d1b154d507b782f84604bcb12/update/sys_script_include_3d46262b97a751106fa8b4b3f153af90.xml:112]: left this in for when the next update to the current contract comes in and we need to support those fields

File: f600233d1b154d507b782f84604bcb12/update/sys_script_include_3d46262b97a751106fa8b4b3f153af90.xml (hunk 2 above)
Comment (tomarnold-ccpo) [reply f600233d1b154d507b782f84604bcb12/update/sys_script_include_3d46262b97a751106fa8b4b3f153af90.xml]: left this in to support the new fields

File: f600233d1b154d507b782f84604bcb12/update/sys_script_include_3d46262b97a751106fa8b4b3f153af90.xml (hunk 1 above)
Comment (jasonburkert-ccpo) [reply f600233d1b154d507b782f84604bcb12/update/sys_script_include_3d46262b97a751106fa8b4b3f153af90.xml:112]: discussed via slack - for very near future

File: f600233d1b154d507b782f84604bcb12/update/sys_script_include_3d46262b97a751106fa8b4b3f153af90.xml (hunk 2 above)
Comment (jasonburkert-ccpo) [reply f600233d1b154d507b782f84604bcb12/update/sys_script_include_3d46262b97a751106fa8b4b3f153af90.xml]: discussed via slack - for very near future

File: f600233d1b154d507b782f84604bcb12/update/sys_script_include_3d46262b97a751106fa8b4b3f153af90.xml (hunk 3 above)
Comment (jasonburkert-ccpo) [reply f600233d1b154d507b782f84604bcb12/update/sys_script_include_3d46262b97a751106fa8b4b3f153af90.xml]: discussed via slack - pls remove getSingleContract()

**Review by jasonburkert-ccpo — CHANGES_REQUESTED** [review]

File: f600233d1b154d507b782f84604bcb12/update/sys_script_include_3d46262b97a751106fa8b4b3f153af90.xml (hunk 3 above)
Comment (jasonburkert-ccpo) [reply f600233d1b154d507b782f84604bcb12/update/sys_script_include_3d46262b97a751106fa8b4b3f153af90.xml]: Fixed in fa4baaaa38fdaec947f64ef0a05f787fe5672134

**Review by jasonburkert-ccpo — APPROVED** [review]

**Pull request merged**